### PR TITLE
Optimises space turf init [2500 blips]

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/69079

**Running on a i5-1135G7**

Without profiling running, which gives inconsistent result I get these:
Without: ~72 seconds
With: ~62 seconds consistently

## Why It's Good For The Game

Apparently this is better for init time.

## Testing Photographs and Procedure
image

## Changelog
:cl:
fix: Speeds up init times by reducing the amount of work to load space turfs.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
image
image

**Comment by PowerfulBacon** [comment]
I've compiled with and without a bunch of times, and the timing of init is very consistent and always better. I am not sure about what is causing the inconsistencies and slower init times with the unit testing.

**Comment by PowerfulBacon** [comment]
The current run was a lot quicker. I'm putting this down to general load on the runners at the time, as the runners have very inconsistent times in general. My results on local are a lot more consistent and consistently better.

File: code/__DEFINES/layers.dm [hunk 1]
```diff
@@ -104,6 +104,10 @@
 #define DATA_HUD_PLANE 15
 
 //---------- LIGHTING -------------
+
+/// Default plane for lighting
+#define LIGHTING_PRIMARY_LAYER 15
```
Comment (itsmeow) [inline code/__DEFINES/layers.dm]: this is unused

**Review by itsmeow — APPROVED** [review]
Not creating thousands of fullbright objects 👍🏻

File: code/__DEFINES/layers.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/__DEFINES/layers.dm]: o yea

File: code/__DEFINES/layers.dm [hunk 2]
```diff
@@ -104,6 +104,10 @@
...
+
```
Comment (PowerfulBacon) [inline code/__DEFINES/layers.dm]:
```suggestion
```

**Comment by itsmeow** [comment]
You can't give yourself good PR points silly

**Comment by PowerfulBacon** [comment]
I'm not giving it to myself, I'm giving it to Kylerace through the transitive property.

**Comment by itsmeow** [comment]
I see.

**Pull request merged**